style.css

```diff
@@ -0,0 +1,138 @@
+body {
+	width: 1440px;
+	height: 1024px;
+	margin: 0;
+	background: url(images/background.png);
+}
+
+.wrap {
+	font-family: Roboto;
+	color: #fff;
+	padding: 129px 69px 104px 103px;
+}
+
+.title {
+	font-size: 57px;
+}
+
+.subtitle {
+	font-size: 19px;
+	line-height: 35px;
+	margin-bottom: 60px;
+}
+
+.label {
+	position: relative;
+}
+
+.text {
+  display: block;
+	font-size: 50px;
+  padding-left: 41px;
+	margin-bottom: 24px;
+}
+
+.text:hover {
+	opacity: 0.8;
+	cursor: pointer;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/evagame/pullRequest?prid=5788637)